sql/2022/sustainability/unminified_css_bytes.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL
+# Histogram of wasted bytes per page
+
+CREATE TEMPORARY FUNCTION getUnminifiedCssBytes(audit STRING)
+RETURNS ARRAY<INT64> LANGUAGE js AS '''
+try {
+  var $ = JSON.parse(audit);
+  return $.details.items.map(({wastedBytes}) => wastedBytes);
+} catch (e) {
+  return [];
+}
+''';
+
+SELECT
+  client,
+  IF(unminified_css_kbytes <= 200, CEIL(unminified_css_kbytes / 10) * 10, 200) AS unminified_css_kbytes,
```

Comment:
I'm not getting this. If it's less than 200 KB then round it up to nearest 10, otherwise just say it's 200 KB? What if it's 4 MB?

Again (sounding like a broken record here!) I think percentiles would be a better, more easily understood, and more consistent with the rest of the Web Almanac approach.

Edit - oh I see you do that in a query below. Not sure what the difference is between these and why you need them both?

Reply:
Rewrote this query similar to the unused one.